lib/project-data.ts

```diff
@@ -87,6 +97,25 @@ export class ProjectData implements IProjectData {
 		this.$errors.fail("No project found at or above '%s' and neither was a --path specified.", projectDir || this.$options.path || currentDir);
 	}
 
+	private getProjectIdentifiers(data :any): IProjectIdentifier {
```

Comment:
since this is a private function, used only once, why don't we change the signature to:
`private getProjectIdentifiers(projectIdentifier :string): IProjectIdentifier`

Reply:
Rosen is right that the argument might be a string and an object. This depends on the use case of the user - if he needs different identifiers for Android and iOS the argument will be object.

Reply:
In this case please rename `data: any` to `projectIdentifierInfo: any` or something better, but let it not be just `data`